tests/ssl-opt.sh

```diff
@@ -9447,7 +9497,7 @@ run_test    "DTLS reassembly: no fragmentation (openssl server)" \
 
 requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
 run_test    "DTLS reassembly: some fragmentation (openssl server)" \
-            "$O_SRV -dtls -mtu 768" \
+            "$O_SRV -dtls -mtu 256" \
```

Comment:
Seems unrelated. Is it an accident or am I missing something?

Reply:
Nope, this is intentional. The test expects `"found fragmented DTLS handshake message"` on the client's output, but with 768 this does not happen. Checking at the proxy log output and also at test `"DTLS reassembly: fragmentation, nbio, renego (gnutls server)"` I thought about reducing that MTU size and that did the trick.

My suspect is that disabling ECDHE and having only ECDH cause the test to be executed (and that is fine, of course), but exchanged messages are somehow smaller in size so that they do not get split into smaller chunks. Howerver this is just my hypothesis, so if I'm doing something really wrong here I will revert that ;)

Reply:
Ah, right, your hypothesis makes sense to me!